app/src/main/java/com/example/woloxapp/ui/login/LoginFragment.kt

```diff
@@ -1,15 +1,25 @@
 package com.example.woloxapp.ui.login
 
 import android.os.Bundle
+import android.view.Gravity
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Toast
 import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.navigation.fragment.findNavController
+import com.example.woloxapp.R
 import com.example.woloxapp.databinding.LoginFragmentBinding
 
 class LoginFragment : Fragment() {
+    private lateinit var loginViewModel: LoginViewModel
     private lateinit var _binding: LoginFragmentBinding
     private val binding get() = _binding
+    companion object {
```

Comment:
companion objects usually go on the bottom of the class
https://kotlinlang.org/docs/coding-conventions.html#class-layout